runtime/lifecycle/lifecycle.go

```diff
@@ -13,15 +13,15 @@ type Lifecycle struct {
 //
 // Example:
 //
-//   if err := client.Prisma.Connect(); err != nil {
-//     handle(err)
-//   }
+//	if err := client.Prisma.Connect(); err != nil {
+//	  handle(err)
+//	}
 //
-//   defer func() {
-//     if err := client.Prisma.Disconnect(); err != nil {
-//       panic(fmt.Errorf("could not disconnect: %w", err))
-//     }
-//   }()
+//	defer func() {
+//	  if err := client.Prisma.Disconnect(); err != nil {
+//	    panic(fmt.Errorf("could not disconnect: %w", err))
+//	  }
+//	}()
```

Comment:
it has to be indented like before, otherwise it'll not render as code in IDEs

Reply:
I've fixed this [lint error](https://github.com/prisma/prisma-client-go/actions/runs/3275369205/jobs/5390123681#step:8:7).

Please refer to the following.
https://tip.golang.org/doc/go1.19#go-doc

Reply:
Ah thanks, my mistake. Then my editor is just not interpreting it right yet.